docs/detections/rules-ui-create.asciidoc

```diff
@@ -318,23 +318,23 @@ Non-aggregating queries doesn't use `STATS...BY` functions and doesn't aggregate
 Here is an example non-aggregating query:
```

Comment:
there are also 2 `[metadata]` operators here:

https://github.com/elastic/security-docs/blob/036ee54e14fc3664e8cec3824b40f1f29e680951/docs/detections/api/rules/rules-api-create.asciidoc?plain=1#L1069

and here

https://github.com/elastic/security-docs/blob/036ee54e14fc3664e8cec3824b40f1f29e680951/docs/detections/api/rules/rules-api-create.asciidoc?plain=1#L1530